website/content/en/docs/contribution-guidelines/testing/local-docs.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Build and serve the docs locally
 linkTitle: Local Docs
+weight: 50
```

Comment:
Move to the Testing Category

Reply:
Personally I would not think to look in the general "testing" area for how to build docs locally. IMO building the docs locally should stay top level in contribution guidelines.

Reply:
reverted